_Console/BuildDevSave/readme.md

```diff
@@ -1,3 +1,10 @@
 This was originally intended to be extended into a general-purpose Lua script reader to have access to QueryCiv3, but the immediate need was to export select Civ3 map data to JSON for temp use in C7-Game/Prototype. After I got Lua working and printing out the info I realized it's simpler just to skip Lua. This is now both acting as a tester for the import Civ3 save function of C7GameData and to set the default/hard-coded map for C7Game.
 
 The script file reference, save file path, and civ3 path are hard-coded for now, and it's currently unclear how long we'll need this, but it might be nice to parameterize this.
+
+#### Build & Use
+
+- Civ3 home is currently hard, coded, and so is relative /Conquests/Saves/for-c7-seed-1234567.SAV 
+- `nuget restore BuildDevSave.csproj`
+- `msbuild`
+- `mono bin/Debug/net472/BuildDevSave.exe`
```

Comment:
Fascinating, I never knew this is how I was supposed to run BuildDevSave.  I've been running it with `dotnet run`, at least since I got the correct targeting pack installed.

I'm going to assume this works on Linux/Mac.  I didn't have either nuget, msbuild, or mono accessible at the command prompt on Windows (I now have nuget, and the first step works), so I can't validate these steps myself.

Reply:
Well, it wasn't being run like this before. I was trying to figure out Mono command line usage. But ... well just look at https://docs.godotengine.org/en/stable/getting_started/scripting/c_sharp/c_sharp_basics.html and the conversation around that in CFC Forum. It's kind of turning everything on its head for me re: .NET Core vs Mono

Reply:
What part of it, exactly?  The Mono 6.x is equivalent to... .NET Framework 4.8 IIRC?  I am thinking, "I should try that external editor setting" after perusing it, as well as that maybe this it the time for me to redeem that JetBrains Rider free trial.